### DISCUSSION TITLE
Github support hasn't answered a ticket that has existed for 4 days now.

### DISCUSSION THREAD

**Discussion by StenSmeets1** [community/community#61942] [GitHub Education]
I have created a ticket to apply to the github education programme because, I cant apply through the normal thing as my school does not give school emails and I keep getting rejected. So 4 days ago I created a ticket for it with a lot of information in it and I even added some more info 2 days ago but I still haven't received any response from support in the ticket. Does anyone know if this is normal to take that long?

**Comment by tuves** [comment]
Welcome to the GitHub Community, @StenSmeets1, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. 

Moreover, I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.


As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.


If you need help with applying please check out our [Apply to GitHub Global Campus for Students](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student) as well as our [Why wasn't my application to Global Campus for students approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved). We aim to assist our students by providing rejection reasons throughout the application process so that you can take steps to rectify and re-apply. But we also understand that there are times where you need a little more guidance and we're here to nudge you in the right direction. Best of luck!

**Reply by StenSmeets1** [reply]
Hello,
Thank you for the response. I have read what my application misses and almost each time the reason is that the school I am enrolled in requires a school email to verify on github. The issue is that the school I am in doesnt give school emails. Furthermore I have tried every option I can with applying and each time sending in the correct and good proof of my enrollment. But each time it still denies me, that is why I created the ticket because at this point I dont know what to do to get it accepted.
The way I know that the proof I send is good, is because with the exact same proof I did get accepted to the Jetbrains education programme. 
So my problem at the moment is just I cant verify here because I do not have an school email, that is why the ticket exists. If you or anyone has any other advice on how to apply please let me know!
